packages/utilities/psammead-rich-text-transforms/package.json

```diff
@@ -29,4 +29,4 @@
   "devDependencies": {
     "jest": "^25.1.0"
   }
-}
+}
```

Comment:
Just missing a new line here

Reply:
hmmmmmm prettier seems to be removing the newline for me

Reply:
fixed this now